runtime/Makefile

```diff
@@ -440,6 +440,10 @@ MAC_LIBS =                   \
 #    This is used linked into the native executable when using the system Mono (i.e. not)
 #    embedding the Mono runtime).
 #
+# libxamarin-coreclr.a:
```

Comment:
`libxamarin-dotnet-coreclr.a` ?

Reply:
Yes, that's better :smile: I'll fix it in a different PR.